noir-projects/noir-contracts/contracts/docs/docs_example_contract/src/main.nr

```diff
@@ -123,8 +123,8 @@ pub contract DocsExample {
         // docs:start:state_vars-PrivateMutableGet
         let card = storage.legendary_card.get_note().note;
         // docs:end:state_vars-PrivateMutableGet
-        let points = card.points + 1;
-        let new_card = CardNote::new(points, card.randomness, context.msg_sender());
```

Comment:
Refactored CardNote here such that we don't need to pass in randomness which allows me to then not expose randomness on the struct here.